### DISCUSSION TITLE
Set-PnPListItem -UpdateType option "SystemUpdate" and "UpdateOverwriteVersion" behaviors are not match the document.

### DISCUSSION THREAD

**Discussion by rioviejo-vivo** [pnp/powershell#3859] [Q&A]
Hello, this is my first post. 
Now I suspect the document is mis-described. But I'm not sure. What would be the expected behaviors?

On the document about Set-PnPListItem,  
https://pnp.github.io/powershell/cmdlets/Set-PnPListItem.html
for  -UpdateType option 'SystemUpdate', it described 
"Sets field values and does not create a new version. **Any events on the list will trigger**. The "Modified By" and "Modified" fields not updated and can not be set." .
However  a flow, which has ”When an item is created or modified” trigger, was **not** triggered when  I excuted  Set-PnPListItem cmdlets with -UpdateType "SystemUpdate".
For  -UpdateType option 'UpdateOverwriteVersion', it described 
"Sets field values and does not create a new version. **No events on the list will trigger**.  **The "Modified By" and "Modified" fields are not updated** but can be set by passing the field values in the update. HINT: use 'Editor' to set the "Modified By" field.".
However  a flow, which has ”When an item is created or modified” trigger, was **triggered** when I excuted  Set-PnPListItem cmdlets with -UpdateType "UpdateOverwriteVersion". And the "Modified By" and "Modified" fields are **always updated** without explicitly setting the field values.

My environment is PowerShell 7.4.1 and PnP.PowerShell 2.4.0.

**Comment by paul-il779** [comment]
When minor versions are enabled on a list, UpdateOverwriteVersion does not behave as documented. Running:
`Set-PnPListItem -List "Documents" -Identity 1 -Values @{"Title" = "Test"} -UpdateType UpdateOverwriteVersion`
will increment the minor version, even though UpdateOverwriteVersion is supposed to prevent versioning changes.
